### PR TITLE
Push validator updates: extensions dir split and track in amp-audio, amp-video.

### PR DESCRIPTION
**Pull request by powdercloud**
Release notes:
- Allow `<track>` tag inside `<amp-audio>`, `<amp-video>`. Require that `<track>` contain a `srclang` attr if `kind=subtitles` (Github #2731)

### CODE REVIEW

**Comment by powdercloud** [comment]
I can only assign one reviewer here, but I'll wait til you're happy as well, @dvoytenko and @honeybadgerdontcare.

**Comment by cramforce** [comment]
:100: WOW, AWESOME!

File: extensions/amp-brid-player/0.1/test/amp-brid-player.out [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FAIL
```
Comment (dvoytenko) [inline extensions/amp-brid-player/0.1/test/amp-brid-player.out]: Should we also name these `validator-amp-brid-player.out`? Or were you planning to do this later?

**Comment by dvoytenko** [comment]
@powdercloud LGTM from my end with one naming question.

File: extensions/amp-brid-player/0.1/test/amp-brid-player.out (hunk 1 above)
Comment (powdercloud) [inline extensions/amp-brid-player/0.1/test/amp-brid-player.out]: This is done, thank you.

**Pull request merged**